src/Components/Navigation.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import { css, withStyles } from "../withStyles";
+
+import FlexContainer from "../Containers/FlexContainer";
+import List from "../Elements/List";
+import Icon from "../Elements/Icon";
+import ListItem from "../Elements/ListItem";
+import Heading from "../Elements/Heading";
+
+const Navigation = ({ styles, ...props }) => (
+  <aside {...css(styles.sidebar)} {...props}>
+    <div {...css(styles.logoArea)}>
+      <Heading {...css(styles.logo)} size="2">
+        Feed<span>.io</span>
+      </Heading>
+    </div>
+    <span {...css(styles.divider)} />
+    <FlexContainer align="start">
+      <ul {...css(styles.dashboardNav)}>
```

Comment:
Use ListItem and List component instead? Otherwise it looks good!

Reply:
I opted against using those due to this being a very specific edge case. Using general components in this instance would introduce more headaches, rather than not.